BUILDING

```diff
@@ -6,8 +6,17 @@
     go get -u golang.org/x/net/context/ctxhttp
     go get -u github.com/golang/protobuf/proto
     go get -u github.com/golang/protobuf/protoc-gen-go
+    
+3. Install drone
 
-3. Install binaries to $GOPATH/bin
+    go get -u github.com/drone/drone/...
+
+4. Install binaries to $GOPATH/bin
 
     go install github.com/drone/drone/cmd/drone-agent
     go install github.com/drone/drone/cmd/drone-server
+
+5. Test
+
+   cd $GOPATH/src/github.com/drone/drone
+   go test -cover $(go list ./... | grep -v /vendor/)
```

Comment:
This actually changes with Go 1.10. The `go test` command excludes the vendor directory automatically. So the syntax could be simplified to this:

```
go test github.com/drone/drone/...
```

Reply:
The current build for the project still use `1.9`, I pulled this line straight from `.drone.yml`